packages/charts/react-charting/src/components/LineChart/LineChart.base.tsx

```diff
@@ -879,23 +1000,59 @@ export class LineChartBase extends React.Component<ILineChartProps, ILineChartSt
         }
 
         const isPointHighlighted = this.state.activeLine !== null && this.state.activeLine === i;
-
-        pointsForLine.push(
-          <circle
-            id={`${this._staticHighlightCircle}_${i}`}
-            key={`${this._staticHighlightCircle}_${i}`}
-            r={5.5}
-            cx={0}
-            cy={0}
-            fill={theme!.semanticColors.bodyBackground}
-            strokeWidth={DEFAULT_LINE_STROKE_SIZE}
-            stroke={lineColor}
-            visibility={isPointHighlighted ? 'visibility' : 'hidden'}
-            onMouseMove={this._onMouseOverLargeDataset.bind(this, i, verticaLineHeight, yScale)}
-            onMouseOver={this._onMouseOverLargeDataset.bind(this, i, verticaLineHeight, yScale)}
-            onMouseOut={this._handleMouseOut}
-          />,
-        );
+        if (!this.props.optimizeLargeData) {
```

Comment:
>optimizeLargeData [](http://example.com/codeflow?start=24&length=17)

the block where optimizeLargeData is false is already present below. Why adding here again